.circleci/config.yml

```diff
@@ -1068,9 +1067,6 @@ workflows:
       - test-core3:
           requires:
             - build-linux
-      - test-wasm64:
```

Comment:
Doing this will cause the wasm64 libraries to be built (redundantly) on the tester rather than the builder, right? is that what we want?

Reply:
No, doing this simple removes test-wasm64 from the list of jobs to run.